app/views/layouts/_header.html.erb

```diff
@@ -0,0 +1,14 @@
+<header class="navbar navbar-fixed-top navbar-inverse">
+  <div class="container">
+    <%= link_to "sample app", '#', id: "logo" %>
```

Comment:
Qestion: `sample app` chỗ này viết thường mà sao a thấy chụp màn hình lại in hoa hết e hè

Reply:
do có id logo
mà id logo trong file custom.scss có text-transform là uppercase nên văn bản trong logo bị in hoa hếtttt